src/components/dataset/query/sidebar/panels/DatasetAccess.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Link, Typography } from '@material-ui/core';
+import UserList from '../../../../UserList';
+import { DATASET_ROLES } from '../../../../../constants';
+import { getRoleMembersFromPolicies } from '../../../../../libs/utils';
+
+const styles = (theme) => ({
+  root: {
+    display: 'block',
+    margin: theme.spacing(1),
+  },
+  paperBody: {
+    padding: theme.spacing(2),
+  },
+  headerText: {
+    fontSize: '14px',
+    lineHeight: '22px',
+    fontWeight: '600',
+    marginTop: theme.spacing(3),
```

Comment:
is this being used anywhere? also, we have some preset text styling in theme.js that might be worth looking at